Getting-Started/Setup/Upgrading/general.md

```diff
@@ -60,9 +60,9 @@ There's also the possibility that some files in the /Config folder are new or so
 Up until version 6.0.0 it was necessary to change the version number in ClientDependency.config. This was to clear the cached html/css/js files in the backoffice. Just change the current version number to one that's higher than that. Make sure not to skip this step as you might get strange behaviour in the backoffice otherwise.
 
 ## Merge UI.xml and language files
-Some packages (like Contour) add dialogs to the UI.xml. Make sure to merge those changes back in from your backup during the upgrade so that the packages continue to work. This file can be found in: /Umbraco/Config/Create/UI.xml.
+Some packages (like Umbraco Forms) add dialogs to the UI.xml. Make sure to merge those changes back in from your backup during the upgrade so that the packages continue to work. This file can be found in: /Umbraco/Config/Create/UI.xml.
```

Comment:
I think this is incorrect.  Umbraco Forms doesn't add it in that file.  On the other hand, Umbraco Contour did.  We can't just replace the wording here.